templates/components/products/price-range.html

```diff
@@ -33,8 +40,31 @@
         {{/if}}
     </div>
 {{/and}}
+{{#and price.retail_price_range.min.without_tax price.retail_price_range.max.without_tax}}
```

Comment:
I think you might want this above the `price_range` html, that way it would appear where the other retail price range appears.

Reply:
I moved the `without_tax` range (and other without tax elements) below the with tax elements to make sure the order of these elements does not differ between the use case of having a price range on initial page load vs having a single price. On the single price display (seen in `prices.html`) we also have all of the with tax elements above the without_tax elements in a descending order of retail price, non-sale price, and sale price (or price).  I believe the way I have this page ordered will ensure that both codepaths produce the same order of displaying prices.